content/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue.md

```diff
@@ -37,7 +43,7 @@ You can manually link up to ten issues to each pull request. The issue and pull
 
 ### Linking a pull request to an issue using a keyword
 
-You can link a pull request to an issue by using a supported keyword in the pull request's description.
+You can link a pull request to an issue by using a supported keyword in the pull request's description or in a commit message (please note that the PR must be on the default branch). 
```

Comment:
This looks like a good solution 😄 Thanks 🙇🏻‍♀️ 

We generally write out "pull request" in full (although I can see a few topics do use PR).

```suggestion
You can link a pull request to an issue by using a supported keyword in the pull request's description or in a commit message (please note that the pull request must be on the default branch). 
```